.github/CODEOWNERS

```diff
@@ -37,6 +37,7 @@
 
 # Translations
 airflow-core/src/airflow/ui/src/i18n/locales/zh_TW/ @Lee-W
+airflow-core/src/airflow/ui/src/i18n/locales/ko/ @choo121600
```

Comment:
Can you please separate this code owner change to its own PRs.

Most likely there will be discussion and we still need to find a process on how to nominate such people from the community.

Thank you for your understanding.

Also I don't think you can be code owner without write permission to the project.

Reply:
Yes, I understand. I have removed the code owner in the Korean translation in this PR. 
I will leave the code owner to contributor who is authorized to write.